target/linux/lantiq/base-files/etc/board.d/01_leds

```diff
@@ -53,6 +53,9 @@ bt,homehub-v5a)
 netgear,dm200)
 	ucidef_set_led_netdev "lan" "lan" "dm200:green:lan" "eth0"
 	;;
+avm,fritz7312)
```

Comment:
This shouldn't be required. Due to the `led-wifi` alias in the dts, the `phy0tpt` trigger should be used by default.